hw2/fastq_filtrator.py

```diff
@@ -0,0 +1,96 @@
+# This function returns GC-content of sequence in percents
+def gc_content(read_seq):
+    return sum(map(read_seq.upper().count, ['G','C'])) / len(read_seq.strip()) * 100
+
+# This function checks whether GC-content of sequence is in bounds 
+def gc_filter(read_seq, gc_bounds):
+    if isinstance(gc_bounds, (tuple, list)):
+        min_gc, max_gc = gc_bounds
+    else:    # Only upper bound is given
+        min_gc = 0
+        max_gc = gc_bounds
+    if gc_content(read_seq) >= min_gc and gc_content(read_seq) <= max_gc:    # GC-content is in bounds
+        return 'Pass'
+    else:
+        return 'Fail'
```

Comment:
В подобных случаях не стоит возвращать строки, для этого идеально подходят `True` и `False`. Тогда это ещё можно будет сильно упростить
```suggestion
    return min_gc <= gc_content(read_seq) <= max_gc:    # GC-content is in bounds
```